markdown/2017-10-23.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Weekly Digest
+show_social: true
+show_agenda: true
+show_sponsorship: true
+...
+
+---
+
+<center>
+<div style="position:relative;width:50%">
+    ![Want want biscuit](https://i.imgur.com/VTZIP2c.png "I didn't know this biscuit existed until today. Interesting combination of salty and sweet, with a rice-y aftertaste. 8/10 would eat again.")
+</div>
+</center>
+
+---
+
+Hi all,
+
+Welcome back to yet another busy week full of events and coursework!
+
+Consistent with previous weeks, this week we bring you a whole _two_ events.
+First a workshop run by Next Jump tomorrow (Tuesday), and a G-Research coding
```

Comment:
delete "tomorrow"

Reply:
but it's being sent out today? Hmm maybe should delete